protob/messages.proto

```diff
@@ -913,6 +922,67 @@ message CosiSignature {
 	optional bytes signature = 1;		// Signature
 }
 
+//////////////////////
+// Stellar messages //
+//////////////////////
+
+/**
+ * Request: Public key at the specified index
+ * @next StellarPublicKey
+ */
+message StellarGetPublicKey {
+	required uint32 index = 1;			// Account index (derived as m/44'/148'/index')
```

Comment:
Use `optional` instead of `required` .

The rationale is explained in https://developers.google.com/protocol-buffers/docs/proto#specifying-field-rules under "Required Is Forever". Furthermore, Proto3 has removed `required`.